Source/WebKit/UIProcess/WebFrameProxy.h

```diff
@@ -214,6 +214,7 @@ class WebFrameProxy : public API::ObjectImpl<API::Object::Type::Frame>, public C
     void updateOpener(WebCore::FrameIdentifier);
     WebFrameProxy* opener() { return m_opener.get(); }
     void disownOpener() { m_opener = nullptr; }
+    RefPtr<WebFramePolicyListenerProxy> activeListener() { return m_activeListener; }
```

Comment:
This is never set.

Reply:
It is set in `WebFrameProxy::setUpPolicyListenerProxy`